tests/unit_test/data_processor/data_processor_test.py

```
@@ -7,13 +7,16 @@
 import urllib
 from datetime import datetime, timedelta, timezone
 from io import BytesIO
+from pathlib import Path
 from typing import List
 from urllib.parse import urljoin
```

Ruff (0.8.2)

`urllib.parse.urljoin` imported but unused

Remove unused import: `urllib.parse.urljoin`

(F401)